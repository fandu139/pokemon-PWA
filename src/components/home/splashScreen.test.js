import React from "react";
import Render from 'react-test-renderer';
import SpalashScreen from './splashScreen';

jest.mock('react-router-dom')

describe('Snapshoot', () => {
  test('Home Loading', () => {
    const component = Render.create(<SpalashScreen />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
