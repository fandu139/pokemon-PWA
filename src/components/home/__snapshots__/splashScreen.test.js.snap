// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshoot Home Loading 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <div
      className="sc-bdVaJa haSTJE sc-htpNat huMSff"
    >
      <div
        className="lazyload-placeholder"
        style={
          Object {
            "height": undefined,
          }
        }
      />
    </div>
  </header>
</div>
`;
